.devcontainer/community_dev/devcontainer.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Bitwarden Community Dev",
+  "dockerComposeFile": "../../.devcontainer/bitwarden_common/docker-compose.yml",
+  "service": "bitwarden_server",
+  "workspaceFolder": "/workspace",
+  "customizations": {
+    "vscode": {
+      "settings": {},
+      "features": {},
+      "extensions": ["ms-dotnettools.csharp"]
```

Since I sat on this to long the C# Dev Kit has come out and we should probably use that instead. 
```suggestion
      "extensions": ["ms-dotnettools.csdevkit"]
```